data/856/337/93/85633793.geojson

```diff
@@ -1741,6 +1742,7 @@
         "wd:id":"Q30",
         "wmo:id":"US"
     },
+    "wof:concordances_official":"iso:code",
```

Comment:
Hmm so in this case, the `uscensus:geoid` does exist for the United States, but we're not calling that out as the official concordance?

Just trying to understand the usage here.. seems to me like _all_ of the records in this repo should get a `wof:concordances_official`  with "uscensus:geoid" as the value and `wof:concordances_official_alt` with "iso:code" as the value.

Reply:
You found an edge case! So the US is the only country with data in WOF where the NMA/NSA/Census describes the whole country as having a code. Probably other countries do this too IRL, but I didn't see it in our data. 

For other placetypes the peers are all _in-country_ so using the NMA/NSA/Census makes logical sense.

But for country/dependency placetypes all the peers are _out-of-country_, so it made more sense to me to always use `ISO:code` for that (and in some cases I set a `ISO:code_pseudo` like for Kosovo).

NOTE: Since the _value_ of the United States official geoid and the ISO code are the same (`US`), this amounts to a no-op to end users.

I'll add this to the main issue's comment...